test/functional/pdo_sqlsrv/pdo_aev2_keywords.phpt

```diff
@@ -0,0 +1,54 @@
+--TEST--
+Test ColumnEncryption values.
+--DESCRIPTION--
+This test checks that connection fails when ColumnEncryption is set to nonsense,
+or when it is set to a bad protocol. Then it checks that connection succeeds when
+the attestation URL is bad.
+--FILE--
+<?php
+include("MsSetup.inc");
+include("AE_v2_values.inc");
+include("pdo_AE_functions.inc");
+
```

Comment:
`require_once`  + skipif
re your comment above, can you find a better word for `bad` to describe the URL?